src_assets/common/assets/web/apps.html

```diff
@@ -302,6 +302,21 @@ <h4 class="modal-title">Covers Found</h4>
           must be a PNG file. If not set, Sunshine will send default box image.
         </div>
       </div>
+      <div class="env-hint">
+        <div class="form-text"><b>About Environment Variables: </b> All commands get these environment variables by default: </div>
+        <table>
+          <tr><td><b>Var Name</b></td><td><b></b></td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_APP_ID</td><td>App ID</td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_APP_NAME</td><td>App Name</td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_CLIENT_WIDTH</td><td>The Width requested by the client</td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_CLIENT_HEIGHT</td><td>The Height requested by the client</td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_CLIENT_FPS</td><td>The FPS requested by the client</td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_CLIENT_HDR</td><td>(true/false) if HDR is enabled by the client</td></tr>
+          <tr><td style="font-family: monospace">SUNSHINE_CLIENT_GCMAP</td><td>(int) the requested gamepad mask</td></tr>
```

Comment:
We probably want to give the user a little more than "the requested gamepad mask" for documentation of the GCMAP parameter. Maybe it makes sense to just expose a gamepad count instead?